csrc/executor_params.cpp

```diff
@@ -32,6 +32,46 @@ void LaunchParams::assertValid() {
       gdimz() > 0 && gdimz() <= 65535,
       "Invalid number of blocks in z direction: ",
       gdimz());
+
+  if (dev_prop->clusterLaunch) {
```

Comment:
Does this require CUDA 12?

Reply:
Requires at least 11.8

Reply:
Should we add some macros to skip this piece of code on earlier versions?